shortcut_composer/core_components/controllers/node_controllers.py

```diff
@@ -18,20 +18,20 @@ def refresh(self):
 
 class LayerOpacityController(NodeBasedController):
     """
-    Gives access to active layers' `blending mode`.
+    Gives access to active layers' `opacity` in %.
 
-    - Operates on `BlendingMode`
-    - Defaults to `BlendingMode.NORMAL`
+    - Operates on `integer` in range `0 to 100`
+    - Defaults to `100`
     """
 
     default_value: int = 100
 
     def get_value(self) -> int:
-        """Get currently active blending mode."""
+        """Get current brush opacity."""
```

Comment:
chyba layer zamiast brush

Reply:
Chyba na pewno :D (to twój stary błąd hihihi :D)

Reply:
Ok, turns out there were even more bugs in this file comments. I think I fixed all of them.